scripts/seed.py

```diff
@@ -137,6 +138,8 @@ def create_challenge(title, start_date, end_date, host_team):
     evaluation_script = open(
         os.path.join(settings.BASE_DIR, 'examples', 'example1', 'sample_evaluation_script.zip'), 'rb')
     queue = ''.join(random.choice(string.ascii_letters) for _ in range(75))
+    year = datetime.date.today().year
+    slug = '{t}-{y}'.format(t=title, y=year)
```

Comment:
Please transform `t=title` as : 
- Lower cased Characters 
- Space replaced with `-`

Reply:
I couldn't understand first change you mentioned. Also should I add space for:
{t}-{y} instead of current -?

Reply:
- > I couldn't understand first change you mentioned.

  Slug should be in lower case.
- >Also should I add space for:
{t}-{y} instead of current -

  If `title = ABC Challenge`, then slug using `slug = '{t}-{y}'.format(t=title, y=year)` will be `ABC Challenge-2019`. We don't want this as slug.
A slug is a part of the URL when you are accessing a resource. For example, in 
  ```
  https://example.com/example-resource
  ```
  `example-resource` is a slug, in which:
    - All characters are in lower case
    - spaces are replaced with `-`, as this might be used in URL

  So, for `title = ABC Challenge`, slug can be `abc-challenge-2019`.
 
Let me know if you have any questions :)

Reply:
Thank you @vkartik97 for your detailed answer :smile:  :+1: